scripts/jenkins/Jenkinsfile.release

```diff
@@ -7,6 +7,9 @@ properties([
         string(name: 'TAG',
                defaultValue: 'v0.0.0',
                description: 'CLUO tag to release'),
+        booleanParam(name: "PUSH_LATEST",
```

Comment:
I see. While he's right that many users *want* to deploy latest and I think we *should* require those users to pick an actual hash tag.

Long ago (1.3.X era), a Tectonic release included a component with a "latest" tag by accident. That image then had to be whitelisted and never changed again to avoid breaking clusters at that release. In postmortem, we decided to stop publishing images using "latest" to avoid risk of this happening again. Clusters were to use pinned image versions for best practice, to avoid breaks, so we could freely iterate forward, and we'd receive better bug reports.

I tend to agree with that conclusion and favor not having it.